neocore/IO/BinaryReader.py

```diff
@@ -43,23 +42,14 @@ def unpack(self, fmt, length=1):
         """
         return struct.unpack(fmt, self.stream.read(length))[0]
 
-    def ReadByte(self, do_ord=True):
+    def ReadByte(self):
```

Comment:
this now possibly raises a `ValueError`. Please add the docstring

Reply:
I found  2 places in `neo-python` that changed the default behaviour of `do_ord`.
https://github.com/CityOfZion/neo-python/blob/fe90f62e123d720d4281c79af0598d9df9e776fb/neo/VM/ExecutionEngine.py#L1001
https://github.com/CityOfZion/neo-python/blob/fe90f62e123d720d4281c79af0598d9df9e776fb/neo/Core/State/UnspentCoinState.py#L87

Fortunately both cases are covered by our tests:
https://coveralls.io/builds/21643551/source?filename=neo/VM/ExecutionEngine.py#L1001
https://coveralls.io/builds/21643551/source?filename=neo/Core/State/UnspentCoinState.py#L87

So after removing the now invalid `do_ord` argument in those lines, a `make test` should confirm it still works as in intended.

Reply:
Great!